deepchecks/tabular/checks/model_evaluation/regression_systematic_error.py

```diff
@@ -37,6 +39,8 @@ def __init__(
         random_state: int = 42,
         **kwargs
     ):
+        warnings.warn('RegressionSystematicError check is deprecated and will be removed in future version,'
```

Comment:
I know we didn't use to do that, but I think it's worth while adding a notice about deprecation (and a link to the new updated better check) in the doc for this check